openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -225,18 +234,22 @@ def test_get_courses_data(self, get_course_mock, get_date_mock, get_salesforce_m
             },
         ]
         now = datetime.now()
+        now_date = now.strftime('%Y-%m-%d')
         course_mock = Mock()
         course_mock.end = now
         course_mock.display_name = 'test-course'
         course_mock.other_course_settings = {'salesforce_data': {}}
         get_course_mock.return_value = course_mock
-        get_date_mock.return_value = now
+        course_key_mock = CourseKey.from_string('course-v1:PX+test-course-run-id+2015')
```

Comment:
I think you can get rid of this variable and just assign the value directly. @ivanvgh

Reply:
You are right, thanks.